modules/checking-mco-status.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="checking-mco-status_{context}"]
+= Checking MCO status
+
+To see the MachineConfigs assigned to your {product-title} nodes and see their status, use the `oc` command:
+
+.Procedure
+. To see the number of machines available on your cluster and the role that each plays, type:
```

Comment:
this is showing the number of nodes per pool, if you have non-mco managed nodes they won't show up here.

Reply:
also machines -> nodes